### PR TITLE
Add/jetpack plans copy test

### PR DESCRIPTION
**Pull request by richardmuscat**
Modified (B) should look like this:

wordpress_com

Not necessarily 100% sure this is the best approach. Feedback super welcome :)

### CODE REVIEW

File: client/lib/plans/constants.js [hunk 1]
```diff
@@ -1180,6 +1243,24 @@ export const FEATURES_LIST = {
...
+			'Schedule time with one of our Happiness Engineers.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_MARKETING_AUTOMATION ]: {
+		getSlug: () => FEATURE_MARKETING_AUTOMATION,
+		getTitle: () => i18n.translate( 'Marketing Automation' ),
+		getDescription: () => i18n.translate(
+			'Schedule all your social media posts in advance.'
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'You can now schedule your social media posts in advance with Jetpack Premium and Professional.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js [hunk 2]
```diff
@@ -983,15 +1042,19 @@ export const FEATURES_LIST = {
 	},
 	[ FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY ]: {
 		getSlug: () => FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
-		getTitle: () => i18n.translate( 'Daily Offsite Backups' ),
+		getTitle: () => i18n.translate( 'Daily Off-site Backups' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Daily off-site backups' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-plans-copy-test)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by fditrapani** [comment]
Some minor comments left but otherwise looks good.

File: client/lib/plans/constants.js [hunk 3]
```diff
@@ -578,12 +625,24 @@ export const PLANS_LIST = {
...
+		],
 		getBillingTimeFrame: () => i18n.translate( 'per month, billed monthly' ),
 		getSignupBillingTimeFrame: () => 'per month', //PLANS A/B TEST: Translate if test passes
 	}
 };
 
 export const FEATURES_LIST = {
+	[ FEATURE_BLANK ]: {
+		getSlug: () => FEATURE_BLANK,
+		getTitle: () => '', //PLANS A/B TEST: Translate if test passes
```
Comment (tyxla) [inline client/lib/plans/constants.js]: We won't actually need to translate this one, so we can remove this comment, right?

**Comment by richardmuscat** [comment]
@fditrapani -- not seeing your comments. am i missing something?

**Comment by fditrapani** [comment]
Sorry @richardmuscat I made the comments and it appended it to a review.

**Comment by richardmuscat** [comment]
@fditrapani & @tyxla -- I have committed the changes you requested but I need your help with two things:
- Filippo: I'm not sure how to resolve the active-tests.js conflicts because I'm worried I'll break running tests.
- Filippo/Tyxla: can one of you please merge and ship the test once the conflict is resolved please?

**Comment by tyxla** [comment]
I've rebased the PR to fix the conflicts @richardmuscat. Happy to land this for you when you and @fditrapani give the green light.

**Comment by richardmuscat** [comment]
Green light from me. I think Filippo should confirm too just in case.

On Aug 10, 2017 9:40 AM, "Marin Atanasov" <notifications@github.com> wrote:

I've rebased the PR to fix the conflicts @richardmuscat
<https://github.com/richardmuscat>. Happy to land this for you when you and
@fditrapani <https://github.com/fditrapani> give the green light.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/Automattic/wp-calypso/pull/16716#issuecomment-321489405>,
or mute the thread

**Comment by fditrapani** [comment]
Looks good to me 🚢

**Pull request merged**